src/example-relay/src/components/Navbar.js

```diff
@@ -1,5 +1,7 @@
 // @flow
 
+/* eslint-disable no-useless-computed-key */
```

Comment:
We could consider setting this rule to warning maybe since flow doesn't work with non-string object properties

Reply:
Yeah, go for it. Maybe completely turning it off? Could you please offload it in a different PR?

Also, would it be possible to improve the description of the relevant commits in this PR? I'd like to keep the quality level high and this is one thing which I believe should exist. Especially while adding a new feature like keyframes.

Reply:
Not even talking about this typo:

```
xs: Add keyframes support
^^
```

😬 😅

Reply:
Sorry 🙈 

I will fix it 😂

Reply:
✔️